### ISSUE TITLE
Making the Celestial Gateway useful

### ISSUE THREAD

**Issue by Alastors** [GTNewHorizons/GT-New-Horizons-Modpack#13330]
### Your GTNH Discord Username

Alastor's Game

### Your Pack Version

whatever latest is.

### Your Proposal

<img width="185" alt="Screen Shot 2023-04-26 at 12 27 19 PM" src="https://user-images.githubusercontent.com/78517796/234641107-da6e7392-b10f-428f-8825-4bc46bdac14f.png">

This would be an arcane crafting recipe, not infusion
4 Void Plates
2 Arcane Stone
1 Arcane Levitator
1 Celestial Pearl
1 Mixed Crystal Cluster

75 of each primal

Likewise in this, I would like to see the Celestial pearl's recipe changed to an alchemy recipe:

1 Mana Pearl
16 Iter
16 Praecantatio
16 Permutatio

### Your Goal

The celestial gate is Intradimensional, not Interdimensional, that being you cant say go to the moon from the overworld, as such, the original recipe is fucking insane, it shouldn't use ichor or anything expensive, it's a glorified travel anchor 

### Your Vision

I envision this to be a thaum alternative to the travel anchor, and if this gets accepted, I will remove the celestial gateway from kami and make it just normal tinkerer

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this change please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members think it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Filiz1** [comment]
Celestial Gateway is a great thing to have multiple bases which are needed since IV (Ichor tier). This is one thing - but a much-needed use for CG and it would be a big mistake to change anything here. It balanced, it has important use - everything's great!

**Comment by Alastors** [comment]
How would it be a mistake, it's specifically only intradimensional, not inter, there are cheaper interdimensional options and this would make it much more useful

**Comment by Alastors** [comment]
It's not balanced at all, it's insanely expensive for ***intra***dimensional transport

**Comment by Ydrec** [comment]
While ichor for it was over gating, the suggested recipe is too cheap, thats like MV recipe at most.

**Comment by Alastors** [comment]
Late mv if anything, we're aiming for early to mid HV in this

**Comment by Ydrec** [comment]
That's like before eio travel anchors

**Comment by Alastors** [comment]
That's the point, thaum is full of items that require lots of research but are unlockable slightly earlier than their non magic counterparts, such as the thirsty tank versus the sterling pump

**Comment by Filiz1** [comment]
I made it on IV and I not need it before. But I need exactly intradimensional transport between bases as it is now.

**Comment by Alastors** [comment]
> I made it on IV and I not need it before. But I need exactly intradimensional transport between bases as it is now.

Functionality is not going to change, it's only going to be made cheaper than before. Like I've said.

I made infusion in Stone Age, it doesn't mean it's good balance

**Comment by Filiz1** [comment]
Why do we need a second travel anchor but make an unreasonably easy recipe for unlimited moving any distance? Isn't that cheating?

**Comment by Alastors** [comment]
I'm going to point out most of the pack already uses ftbutils for waypoints including the main servers, so no it isn't cheating

**Comment by Filiz1** [comment]
What about horses? A good transport for MV. Horses and Celestial Gateway not on the same stage a little bit :)

**Comment by Alastors** [comment]
Sir, I don't know what you're smoking but Christ I need me some

**Comment by Dream-Master** [comment]
You know i am always against cheap eraly game travel. So be careful how and if you buff it. @Alastors

**Comment by Alastors** [comment]
<img width="389" alt="Screen_Shot_2023-04-27_at_5 48 11_PM" src="https://user-images.githubusercontent.com/78517796/234999071-db2ff650-4966-482d-b838-5a16cb4f2885.png">
After much deliberation this was the final result, 

That is:
96 iter,
64 alienis,
32 praecantatio,
32 permutatio,
16 desidia
2 pulsating crystals, 
2 shadow metal plates,
2 void metal plates,
1 portable hole,
1 celestial pearl,
1 melding mirror,
1 enderium plate,
1 astral silver frame,
1 mana steel plate.
Center item: Transvector Dislocator


I will follow this up with the proposed change for the dislocator and the pearls as well.

**Comment by Alastors** [comment]
<img width="283" alt="Screen Shot 2023-04-27 at 6 27 34 PM" src="https://user-images.githubusercontent.com/78517796/235004416-ad4f2b9d-f7be-4928-97df-bf5f206c2981.png">
here is the proposed change for the celestial pearl

2 conflux disc
1 Astral Silver Plate
1 Infused Plate
1 Mana Pearl
25 of each primal?

Result: 1 celestial pearl

<img width="297" alt="Screen Shot 2023-04-27 at 6 28 35 PM" src="https://user-images.githubusercontent.com/78517796/235004569-5335f120-ac03-4edd-94ff-197b7413b073.png">
For the new transvector recipe, remove it from using the interface
Basically the same as the original dislocator recipe, but switches out the interface with a levitator (because it makes more sense)
The only other change here is switching the LV circuits for MV circuits, but we could very well switch those out with HV circuits

**Comment by Alastors** [comment]
@Dream-Master could we get any comment on this proposed change?

**Comment by Rakdarian** [comment]
Isnt the Celestial Gateway as is only gated by the ichor(not ichorium) infusion and associated research?
Thats like Mv realistically. Especially considering your soap addition making the warp a non issue. This would be a pretty aignificant nerf unless for some reason you think killing pigmen/endermen is somehow difficult and more so than making mv/hv circuits

**Comment by Alastors** [comment]
Lmfao

**Comment by Chaoschaot232** [comment]
I think it's a two edged sword what you are requesting @Alastors and I can understand why many commentors here are against such buff and why @Dream-Master doesn't accepts cheap traviling recipes.

Sure there are arguments for a buff for it but only a slightly one if accebtable but for now I wouldn't touch it in any way. Also this pack is intended to be hard and painfull and sure there are edges which are discussable but this one looks even for me just fine :)

**Comment by Alastors** [comment]
> I think it's a two edged sword what you are requesting @Alastors and I can understand why many commentors here are against such buff and why @Dream-Master doesn't accepts cheap traviling recipes.
> 
> Sure there are arguments for a buff for it but only a slightly one if accebtable but for now I wouldn't touch it in any way. Also this pack is intended to be hard and painful and sure there are edges which are discussable but this one looks even for me just fine :)

Respectfully, I do not work with GTNH anymore, I'll note however since you took the time for this.

The problem isn't that this couldn't be good at its current state, it could, if it were ***inter***dimensional, but it is ***intra***dimensional.

The issue with that is that there are many other interdimensional movement options that are much cheaper and tiered down in comparison. Take for example the Planar Vortex or even Blood Magic, both of these do both inter and intra but are much more accessible than Kami as a whole is. Likewise, there are ender IO solutions as well as solutions originating out of AE2, so unfortunately this one is sort of the odd-man-out.
